templates/common/infra/bicep/core/host/functions-dotnet.bicep

```diff
@@ -0,0 +1,36 @@
+param environmentName string
+param location string = resourceGroup().location
+
+param allowedOrigins array = []
+param applicationInsightsName string = ''
+param appServicePlanId string
+param appSettings object = {}
+param keyVaultName string = ''
+param linuxFxVersion string = 'DOTNET-ISOLATED|7.0'
```

Comment:
I know I said don't do this before, but I just got a function project that wasn't dotnet isolated, so let's create this file as 'functions-dotnet-isolated.bicep'

Reply:
renamed!